Automattic/abacus
# Add config profiles

_Split from #249_

This PR adds config and config profiles:
- Consolidates static configuration throughout the app into one file
- Allows having different profiles
- Add switch to change to Production config in development mode, add environment variable to set the switch 
- Add warning banner for working with production.
- Ensures development profile and `NODE_ENV=test` for e2e testing





## How has this been tested?






- Automated tests that cover added/changed functionality
- Manual testing with production data (locally)
- Manual testing with mock data (locally or on Vercel)
- Additional manual testing instructions (please specify):
   Try out prod-config-in-development:
   1. `NEXT_PUBLIC_PRODUCTION_CONFIG_IN_DEVELOPMENT=true npm run dev`
   2. Go into the dev tools and delete our auth token in local storage - you should only have to do this once.
